### PR TITLE
Fix/1553 rtt cb not found

### PR DESCRIPTION
**Pull request by tdasika**
pyocd is always searching for control_block_id with 16 bytes length (when the block id is less than 16 bytes, it is padding with ‘\0’)
So, when ‘SEGGER RTT’ is the block id to be found, it is searching for 4 words (16 bytes) with zero padding instead of 10 bytes(’SEGGER RTT’),  where some junk characters may be present and resulting in failure ‘Control block not found'

Fixing it by searching for only the length of the control block ID provided while setting up rtt

### CODE REVIEW

File: pyocd/debug/rtt.py [hunk 1]
```diff
@@ -436,20 +433,20 @@ def _find_control_block(self) -> Optional[int]:
         offset: int = 0
 
         while search_size:
-            read_size = min(search_size, 32)
-            data = self.target.read_memory_block32(addr, read_size)
+            read_size = min(search_size, 8)
```
Comment (flit) [inline pyocd/debug/rtt.py:436]: Reducing the read size will slow the search down noticeably. 
It's probably best to leave this at 32 for now.

(Really, the read size should be at least 512 or 1024 bytes for best performance, especially with HS USB probes. However, with large sizes like that you need to take into account the greater likelihood of running off the end of the memory region when not using the default RAM region.)

File: pyocd/debug/rtt.py [hunk 2]
```diff
@@ -421,10 +421,7 @@ def __init__(self, target: SoCTarget, address: int = None,
             self._cb_search_size_words = 0
         else:
             self._cb_search_size_words = size // 4
```
Comment (flit) [inline pyocd/debug/rtt.py:423]: `self._cb_search_size_words` is set to the size in words here, but it's used as a byte count for `.read_memory_block8()` in the search routine.

So, please remove the divide by 4 and rename the attribute to `_cb_search_size_bytes`.

**Review by flit — CHANGES_REQUESTED** [review]
Just a couple small changes.

If you feel like it, please also add your copyright to the file's header comment.

**Comment by flit** [comment]
@tdasika Do you mind if I commit the requested changes to your branch so I can close out this PR? I'd like to make sure this fix is included in the next release. Thanks!

**Comment by tdasika** [comment]
> @tdasika Do you mind if I commit the requested changes to your branch so I can close out this PR? I'd like to make sure this fix is included in the next release. Thanks!

@flit , I have addressed the requested changes.

**Comment by flit** [comment]
> I have addressed the requested changes.

Even better! Thanks 😄

**Review by flit — APPROVED** [review]

**Pull request merged**